Soccar [Game-Server]/src/nl/soccar/gameserver/model/session/GameWrapper.java

```diff
@@ -35,13 +37,22 @@
     private final GameEngine engine;
     private Timer timer;
 
+    /**
+     * Intializes the GameWrapper class.
+     * 
+     * @param session The session, not null.
+     * @param game The game, not null.
+     */
     public GameWrapper(SessionWrapper session, Game game) {
         this.session = session;
         this.game = game;
 
         engine = new GameEngine(game);
     }
 
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)